### ISSUE TITLE
Any update regarding Magento ver. 2.2.5 version

### ISSUE THREAD

**Issue by ghost** [tig-nl/maxcdn-magento2#2]
**Magento ver. 2.2.5**
We have installed this extension manually at Magento ver. 2.2.5 as it's unable to install via composer in specified version and makes some issues.

**Commands issues**
After installation the module seems not compatible with this version as all the commands using SSH start making issues and after removing the module everything start working fine. 

**Any updated version release ?**
Do you guys have any timeline to update this module to make it compatible with 2.2.5 and latest version of Magento ?

-Zahid

**Comment by schldwcht** [comment]
@tig-support: please test on env.

**Comment by tig-jeffreypeeters** [comment]
Hey @magentogems ,

We would like to make maxcdn available for 2.5.5 and the latest version of magento but we are currently unable to put more time into it. This is mainly because we have a full workload of tasks on our backlog that customers are paying for that need to be resolved before we can work on maxcdn again as it is a project we do for free and thus has a lower priority. This also means we can't give you a indication as for when we would be able to start working on further maxcdn support as we don't know what kind of work could come in down the line.

Kind regards,
Jeffrey Peeters

TIG
